### DISCUSSION TITLE
install bundle https://github.com/ConnectHolland/cookie-consent-bundle/

### DISCUSSION THREAD

**Discussion by mcg-pimcore** [pimcore/pimcore#10305] [Q&A / Support]
Hello, 

As @NiklasBr suggested me to install https://github.com/ConnectHolland/cookie-consent-bundle/  bundle for cookie consent.
I was trying to install this bundle. But some how do not succeed in bundle installation.

I followed instructions to install bundle.
  run this command
       composer require connectholland/cookie-consent-bundle

   then added bundle into composer autoload psr-4
   "ConnectHolland\\CookieConsentBundle\\":"Bundle/CHCookieConsentBundle"
  
then added bundle path in config file.
        bundles: 
        search_paths:
            - bundles/CHCookieConsentBundle
            
 then cache:clear and then 
            composer dump-autoload
  Let me know if there is any other process of doing it or if I followed right instruction then why it is not showed in bundles folder.

**Comment by NiklasBr** [comment]
That bundle is a Symfony bundle, it will never show up in the list of [Pimcore bundles](https://pimcore.com/docs/pimcore/current/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Pimcore_Bundles/index.html) (see [Pimcore docs](https://pimcore.com/docs/pimcore/current/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Bundle_Collection.html) and [Symfony docs](https://symfony.com/doc/current/bundles.html)). But you should probably ask questions about that bundle in the bundle repository.
